plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/discoverer/XcpServerDiscoverer.java

```diff
@@ -536,7 +536,7 @@ private void createXsToolsISO() {
             id = _tmpltDao.getNextInSequence(Long.class, "id");
             VMTemplateVO template =
                     VMTemplateVO.createPreHostIso(id, isoName, isoName, ImageFormat.ISO, true, true, TemplateType.PERHOST, null, null, true, 64, Account.ACCOUNT_ID_SYSTEM,
-                            null, "xen-pv-drv-iso", false, 1, false, HypervisorType.XenServer);
+                            null, "XenServer Tools Installer ISO (xen-pv-drv-iso)", false, 1, false, HypervisorType.XenServer);
```

Comment:
This will create an entry for deployments that do not have it yet using this new "standardized name". However, for environment that already had it from previous versions, the name (display_text) will not be updated, right?

Reply:
That is correct, I can explicitly add `tmplt.setDisplayText("XenServer Tools Installer ISO (xen-pv-drv-iso)");` at line 544 (couple lines down here)

Reply:
I think this is the right thing to do, so we get consistency.
You have to save the `tmplt` as well right? So, you update the data in the database